sources/packages/web/src/composables/useFormatters.ts

```diff
@@ -304,6 +309,23 @@ export function useFormatters() {
     return `${negativeValueDisplay}$${Math.abs(currencyValue).toFixed(2)}`;
   };
 
+  /**
+   * Validates if the given date 1 is before the given date 2.
+   * The comparison is only done with dates, if the date has a
+   * time object, its not considered for the comparison.
```

Comment:
Can we change it to "time object it's not considered for the comparison.", minor but changes too much the meaning IMO.